<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopTab</class>
 <widget class="QWidget" name="DesktopTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsClass1</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="grid">
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="hbox_btn">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="vbox_capture">
      <item>
       <widget class="QCheckBox" name="ck_dxgi">
        <property name="text">
         <string>DXGI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ck_dshow">
        <property name="text">
         <string>DShow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ck_gdi">
        <property name="text">
         <string>GDI</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="hbox_area">
      <item>
       <layout class="QVBoxLayout" name="vbox_area">
        <item>
         <widget class="QCheckBox" name="ck_capture_area">
          <property name="text">
           <string>矩形区域</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ck_full_screen">
          <property name="text">
           <string>全屏</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vbox_edit">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_x"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_y"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>宽度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_width"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>高度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_height"/>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
